content/docs/command-reference/remote/add.md

```diff
@@ -367,6 +367,25 @@ $ dvc remote add -d myremote \
 
 <details>
 
+### Click for OSF
+
+An [Open Science Framework](https://www.cos.io/products/osf) account is needed to use
+[OSF storage](https://help.osf.io/hc/en-us/articles/360054528874). 
+
+```dvc
+dvc remote add myremote osf://osf.io/path/to/dir
+```
+To work with OSF you need to specify several parameters:
+- user
+- project
+- password
```

Comment:
Let's include the `remote modify` commands to add the minimum required credentials instead of the text under the code block, please.

Reply:
I guess like this:
````suggestion
$ dvc remote add myremote osf://osf.io/path/to/dir
$ dvc remote modify osfremote --local project myproject
$ dvc remote modify osfremote --local user example@mail.com
$ dvc remote modify osfremote --local password 12345
```

> The project name, username, and password (may) contain sensitive user info.
> Therefore, it's safer to add them with the `--local` option, so they're
> written to a Git-ignored config file.
````
> Plus added the --local note like in the existing examples.